index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Hello Microverse</title>
+</head>
+<body>
+    <header>
+        <h2 class="heading">Hello Microverse!</h2>
```

Comment:
**Optional**
I will suggest that you use the `H1` tag in the header instead of `H2`.
Why is because H1s are a good indicator of what the most important text on a page is. As they are usually the most visually notable content on the page, a good H1 can ensure clarity for the reader. It's important to remember that SEO can often be about user optimization as well as optimization for search.